profiles/test/data/VP_LUNARG_test_api.json

```diff
@@ -3000,6 +3013,14 @@
                             "VK_PIPELINE_STAGE_2_TOP_OF_PIPE_BIT",
                             "VK_PIPELINE_STAGE_2_DRAW_INDIRECT_BIT"
                         ]
+                    },
+                    "VkQueueFamilyCheckpointPropertiesNV": {
+                        "checkpointExecutionStageMask": [
+                            "VK_PIPELINE_STAGE_TRANSFER_BIT"
+                        ]
+                    },
+                    "VkQueueFamilyQueryResultStatusProperties2KHR": {
+                        "supported": true
```

Comment:
You'll need to add some test JSON with multiple queue families, each with its own set of extension properties to make sure that once you've changed the code to handle separate set of extension properties then they each get their own sets of values.